solutions/standard/main.tf

```diff
@@ -0,0 +1,39 @@
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.4"
+  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
+  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
+}
+
+########################################################################################################################
+# Event Notifications
+########################################################################################################################
+
+locals {
+  service_credential_names = var.service_credential_names
```

Comment:
don't see any reason for creating a local here instead of directly using the var?